packages/perseus/src/widgets/interactive-graphs/interactive-graph-validator.test.ts

```diff
@@ -5,11 +5,28 @@ import {clone} from "../../../../../testing/object-utils";
 import interactiveGraphValidator from "./interactive-graph-validator";
 
 import type {PerseusGraphType} from "../../perseus-types";
-import type {PerseusInteractiveGraphRubric} from "../../validation.types";
+import type {
+    PerseusGraphCorrectType,
+    PerseusInteractiveGraphRubric,
+} from "../../validation.types";
 import type {Coord} from "@khanacademy/perseus";
 
 function createRubric(graph: PerseusGraphType): PerseusInteractiveGraphRubric {
-    return {graph, correct: graph};
+    const correctKeys = [
+        "type",
+        "coords",
+        "center",
+        "radius",
+        "match",
+        "allowReflexAngle",
+    ];
+    const correct = {};
+    for (const key in graph) {
+        if (correctKeys.includes(key)) {
+            correct[key] = graph[key];
+        }
+    }
```

Comment:
Although this is an easy way to build out the `correct` key, I'm not sure it's the best way.  We'll probably want to switch on `graph.type` and then extract the correct set of values for that given graph type (see earlier comment about the `PerseusInteractiveGraphRubric` type).